stylize/__main__.py

```diff
@@ -32,10 +33,21 @@ def current_git_commit():
 ## Yields all files that differ from the branching point with @diffbase or are
 # not tracked by git.
 def enumerate_changed_files(exclude=[], diffbase="origin/master"):
-    # find common ancestor between @diffbase and @current_commit
-    ancestor = subprocess.check_output(
-        ['git', 'merge-base', current_git_commit(), diffbase]).strip().decode(
-            'utf-8')
+    try:
+        # try to find common ancestor between @diffbase and @current_commit
+        ancestor = subprocess.check_output(
+            ['git', 'merge-base', current_git_commit(), diffbase]).strip(
+            ).decode('utf-8')
+    except subprocess.CalledProcessError:
+        # There was no common ancestor between @diffbase and @current_commit
+        ancestor = subprocess.check_output(
+            ['git', 'rev-list', '--max-parents=0', current_git_commit()
+             ]).strip().decode('utf-8')
+        print("[ERR] Your diffbase: '" + str(diffbase) +
```

Comment:
Sorry for the delay on this - I was on vacation for about a week and this completely slipped my mind.

After some thought, I think you're right that it would be best if it logged the message, but continued to run.  However, I think there's an issue with the way it's setup now.  One edge case is if there's only one commit on the current branch.  The commit that gets chosen as the fallback diffbase will be the current commit, resulting in no formatting.

It may be best to just fallback to checking all files if the diffbase doesn't work.  Thoughts?

I just pushed a new test case to this branch highlighting the issue.